luigi/contrib/redis_store.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright 2012-2015 Spotify AB
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import datetime
+import logging
+
+import luigi
+
+logger = logging.getLogger('luigi-interface')
+
+try:
+    import redis
+
+except ImportError:
+    logger.warning("Loading redis module without redis installed")
+
+
+class RedisTarget(luigi.Target):
+
+    """ Target for a resource in Redis."""
+
+    marker_prefix = luigi.configuration.get_config().get('redis',
```

Comment:
Whoups, didn't catch this until now, sorry. But you should use the new configuration system. See this [example](https://github.com/spotify/luigi/blob/b5ad3eba1501bdc25e91e98901bc781128f2d8a7/luigi/worker.py#L217)